examples/experimental/dagster-airlift/dagster_airlift/in_airflow/mark_as_migrating.py

```diff
@@ -61,10 +61,9 @@ def mark_as_dagster_migrating(
 
     for dag in migrating_dags:
         logger.debug(f"Tagging dag {dag.dag_id} as migrating.")
-        dag.tags.append(
-            json.dumps(
-                {"DAGSTER_MIGRATION_STATUS": migration_state.get_migration_dict_for_dag(dag.dag_id)}
-            )
+        Variable.set(
+            key=f"{dag.dag_id}_dagster_migration_state",
```

Comment:
Can you make this a common function in case we have to change things? I'd like to have one canonical spot for dag -> variable name mappign

Reply:
extract out string format into function please

Reply:
ah sorry missed this one.